.gitlab/pipelines/stages/publish.sh

```diff
@@ -2,10 +2,19 @@
 
 set -o errexit -o nounset
 
+version=$(cat "output/version.txt")
+
+if [ -z "${version}" ]; then
+  >&2 echo "Version not set!"
+  exit 1
+fi
+
 mc config host add "s3" "https://s3.amazonaws.com" "${AWS_ACCESS_KEY}" "${AWS_SECRET_KEY}"
 
-mc cp output/kubecf-*.tgz "s3/scf-v3"
-echo "URL: https://scf-v3.s3.amazonaws.com/$(basename output/kubecf-*.tgz)"
+mc cp "output/kubecf.tgz" "s3/scf-v3/kubecf-${version}.tgz"
```

Comment:
We should factor out `scf-v3` and put it somewhere easier to change (since I expect we _will_ want to change it).

But that's totally optional for _this_ PR.

Reply:
I'll do that once we move to Concourse.